sdk/eventhub/event-hubs/README.md

```diff
@@ -10,7 +10,10 @@ The Azure Event Hubs client library allows you to send and receive events in you
 [Product documentation](https://azure.microsoft.com/en-us/services/event-hubs/) |
 [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/eventhub/event-hubs/samples)
 
-**NOTE**: If you are using version 2.1.0 or lower, then please use the below links instead
+**NOTE**: If you are using version 2.1.0 or lower and want to migrate to the latest version
+of this package please look at our [migration guide to move from EventHubs V2 to EventHubs V5](./migrationguide.md)
```

Comment:
Can you make this an absolute URL to the migration guide? I think that's necessary if you're looking at the README from outside GitHub, e.g. NPM.

Reply:
So it's weird, I was told that wasn't necessary because NPM rewrites it properly. Perhaps there are exceptions to this rule.

@sadasant, @willmtemple  - am I wrong about this one?

Reply:
Oh, I didn't know NPM could do that. I noticed that all the other links were absolute so figured that way why.

Reply:
Relative links also help editors navigate through markdowns in source files and during revisions of older versions :) they indirectly keep you in the same branch or git hash.

Reply:
@maggiepint I couldn't find a guideline around whether we should use absolute or relative links in our READMEs. Is one generally preferred over the other?

I did notice that the example README uses absolute links eveywhere:
https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-EXAMPLE.md

Reply:
So, I found we also host the READMEs outside of NPM:
https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.2/index.html
I don't know how a relative path would get resolved here, but this also leads me to think that using an absolute path is probably safer given the README can end up being hosted in different places. The NPM tip is really cool though.

Reply:
@chradek , as usual, makes a compelling point here and I can no longer ignore him and feel like I'm doing something good.

I'm going to swap this over to full URL. Do we think we should do the same in other readmes? (CC: @sadasant , @willmtemple )

Reply:
I find it concerning that we have relative paths in so many places then. Relative paths do help in certain situations. I wonder if we could ask the docs team to add support to relative paths.

Reply:
@KarishmaGhiya  - the docs at this URL are generated by our own doc tool, right?

https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-event-hubs/5.0.2/index.html

Per @sadasant's question - do you know if we can support relative paths in those readmes? Ie, is the readme touched or modified during this process in any way?